code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -75,7 +75,7 @@
 		return
 
 /obj/item/storage/box/survival_vox
-	icon_state = "box_civ"
+	icon_state = "box_vox"

```

Comment:
Lmfao Vox box ftw

Reply:
had to resist the temptation of naming it vox_box and keep it consistent with the rest :(

Reply:
Should have done it anyways

Reply:
All good man, we just gonna read right-left on this one